src/pages/spacewalk/bridge/index.tsx

```diff
@@ -52,10 +52,16 @@ function Bridge(): JSX.Element | null {
     onClick: () => setTabValue(index),
   });
 
+  const bridgeDirection = tabValue === BridgeTabs.Issue ? 'issue' : 'redeem';
```

Comment:
As we use 'issue' and 'redeem' strings several times across all of these files, what about we create an enum `enum BridgeDirections {
ISSUE,
REDEEM
}`

and the use it for the `type BridgeDirection`?